slasher/beaconclient/receivers.go

```diff
@@ -63,8 +63,13 @@ func (bs *Service) receiveAttestations(ctx context.Context) {
 		}
 		if err != nil {
 			log.WithError(err).Error("Could not receive attestations from beacon node")
+			continue
 		}
 		log.WithField("slot", res.Data.Slot).Debug("Received attestation from beacon node")
+		if err := bs.slasherDB.SaveIncomingIndexedAttestation(ctx, res); err != nil {
```

Comment:
Why do we save and then send it to the attestation feed?

Reply:
we save as it comes in, then we send it to the feed in the detection service which will detect for slashable events by looking at minmax spans and then doing db gets for the attestations in the same epoch